src/ray/gcs/gcs_server/test/gcs_worker_manager_test.cc

```diff
@@ -47,8 +45,9 @@ class GcsWorkerManagerTest : public Test {
           new boost::asio::io_service::work(io_service_));
       io_service_.run();
     }));
-    worker_manager_.reset(
-        new gcs::GcsWorkerManager(gcs_table_storage_, gcs_publisher_, kv_instance_));
+    usage_stats_client_.reset(new gcs::UsageStatsClient("127.0.0.1:6379", io_service_));
+    worker_manager_.reset(new gcs::GcsWorkerManager(
+        gcs_table_storage_, gcs_publisher_, *usage_stats_client_));
```

Comment:
Do you mind to change these two to use make_shared/make_unique instead of reset?